netebpfext/net_ebpf_ext_hook_provider.h

```diff
@@ -7,30 +7,33 @@
 #include "framework.h"
 #include "net_ebpf_ext.h"
 
+typedef enum _net_ebpf_extension_hook_attach_capability
+{
+    ATTACH_CAPABILITY_MULTI_ATTACH = 0, ///< Multiple clients can attach to the hook, for each attach parameters.
+    ATTACH_CAPABILITY_SINGLE_ATTACH, ///< Only one client can attach to the hook, irrespective of the attach parameters.
+    ATTACH_CAPABILITY_SINGLE_ATTACH_PER_HOOK, ///< One client can attach to the hook for each attach parameter, but with
```

Comment:
consider rename `ATTACH_CAPABILITY_MULT_ATTACH` and ``ATTACH_CAPABILITY_MULT_ATTACH_WITH_WILDCARD`.